.github/workflows/ci.yml

```diff
@@ -39,4 +39,4 @@ jobs:
       - name: flake8 Lint
         uses: py-actions/flake8@v1
         with:
-          path: "automation.py"
+          path: "restructure_file.py","ES_Upload.py"
```

Comment:
The line should be, one string with the files separated by spaces, be sure to use the full path of your new files, and don't remove the automation.py file from there.